install.sh

```diff
@@ -10,7 +10,14 @@ glibc_filename="glibc-2.31.tar.xz"
 args=("$@")
 
 function build() {
-    (cd "./${prefix}-$1" && makepkg -i "${args[@]}")
+    (
+        cd "./${prefix}-$1"
+        package_filename=$(makepkg --packagelist)
+        if [[ ! -f "$package_filename" ]] ; then
+          makepkg --clean --noconfirm "${args[@]}"
```

Comment:
I think `--noconfirm` only has an effect if `makepkg` is invoking `pacman`, e.g. with `-s` or `-i`.  Since we're not passing those by default I don't think `--noconfirm` needs to be passed by default either.

I'd also rather not have `--clean` by default since sometimes I inspect those directories after a build to figure things out.